src/main/java/com/eternalcode/parcellockers/parcel/ParcelBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.eternalcode.parcellockers.parcel;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import java.util.UUID;
+
+public class ParcelBuilder {
+
+    public static Parcel of(UUID uuid, String senderName, ParcelLocker entryLocker, ParcelLocker destinationLocker, ParcelSize size, boolean priority, String receiver) {
+        Player sender = Bukkit.getServer().getPlayer(senderName);
+        if (sender == null) {
+            return null;
+        }
+        UUID senderUUID = sender.getUniqueId();
+
+        Player recipient = Bukkit.getServer().getPlayer(receiver);
+        if (recipient == null) {
+            return null;
+        }
+        UUID recipientUUID = recipient.getUniqueId();
+        return new Parcel(uuid, senderUUID, new ParcelMeta("Parcel", "", priority, recipientUUID, size, entryLocker, destinationLocker));
+    }
+
+    public static Parcel of(UUID uuid, String senderName, ParcelLocker entryLocker, ParcelLocker destinationLocker, ParcelSize size, boolean priority, String receiver, String name) {
+        Player sender = Bukkit.getServer().getPlayer(senderName);
+        if (sender == null) {
+            return null;
+        }
+        UUID senderUUID = sender.getUniqueId();
+
+        Player recipient = Bukkit.getServer().getPlayer(receiver);
+        if (recipient == null) {
+            return null;
+        }
+        UUID recipientUUID = recipient.getUniqueId();
+        return new Parcel(uuid, senderUUID, new ParcelMeta(name, "", priority, recipientUUID, size, entryLocker, destinationLocker));
+    }
+
+    public static Parcel of(UUID uuid, String senderName, ParcelLocker entryLocker, ParcelLocker destinationLocker, ParcelSize size, boolean priority, String receiver, String name, String description) {
+        Player sender = Bukkit.getServer().getPlayer(senderName);
+        if (sender == null) {
+            return null;
+        }
+        UUID senderUUID = sender.getUniqueId();
+
+        Player recipient = Bukkit.getServer().getPlayer(receiver);
+        if (recipient == null) {
+            return null;
+        }
+        UUID recipientUUID = recipient.getUniqueId();
+        return new Parcel(uuid, senderUUID, new ParcelMeta(name, description, priority, recipientUUID, size, entryLocker, destinationLocker));
+    }
+}
```

Comment:
pododawaj entery, zamiast `Bukkit.getServer()` użyj klasy `Server`. Jeśli ma być to util to klasę zrób finalną oraz dodaj prywatny konstruktor. Szczerze możesz przemyśleć, by przerobić to na service bo nazwa Builder tu nie pasuje

Reply:
Dodałem prywatny konstruktor, a jak mam użyć klasy Server, skoro jest to util i nie mogę przez konstruktor podać mu tej instancji?

Reply:
możesz przez parametr metody to przekazywać